docs/user-guide/howto.rst

```diff
@@ -157,6 +157,28 @@ one for each time bin.
 
 .. accordion-footer::
 
+.. accordion-header::
+    :id: collapseHowToOne
+    :title: Non-convergence of a fit because of IRFs validity range
+
+It happens that a 3D analysis does not converge with warning messages indicating that the
+scanned positions of the model is outside valid IRF map range. The type of warning message is:
+::
+
+    Position <SkyCoord (ICRS): (ra, dec) in deg
+      (329.71693826, -33.18392464)> is outside valid IRF map range, using nearest IRF defined within
+
+This issue might happen when the position of a model has no defined range. The minimizer
+might scan positions outside the spatial range in which the IRFs are computed and then it gets lost.
```

Comment:
I think that the minimizer gets lost because the contribution to the npred of a model located outside is null. The IRF range is a side effect not the cause. e.g. if a region where you have zero exposure, but still a PSF and edisp defined this would fail the same way but without the warning message

Reply:
Really? If everything is defined but with a null exposure, npred will be zero and be compared to the data... Is it a failure case?

Reply:
I think the point is that the `npred` remains zero at all nearby positions, so the fit does not know which direction to follow.